tests/js/spec/views/accountSecurity.spec.jsx

```diff
@@ -232,6 +247,10 @@ describe('AccountSecurity', function () {
     expect(wrapper.find('AuthenticatorStatus').prop('enabled')).toBe(false);
     // user is not 2fa enrolled
     expect(wrapper.find('TwoFactorRequired')).toHaveLength(1);
+
+    expect(wrapper.find('Tooltip')).toHaveLength(1);
+    // expect(wrapper.find('Tooltip').prop('title').children()).toContain('to enable 2FA');
```

Comment:
can't figure out how to check the children of a prop for text 😭 @leedongwei or others have an idea? just really want to make sure this is the correct tooltip its finding

Reply:
You could use

```javascript
const tooltip = mountWithTheme(wrapper.find('Tooltip').prop('title'));
expect(tooltip.text()).toContain('to enable 2FA');
```

Reply:
⭐ awesome thx!!